mycroft/api/__init__.py

```diff
@@ -16,7 +17,17 @@ def __init__(self, path):
         self.version = config_server.get("version")
         self.identity = IdentityManager.get()
 
-    def request(self, params):
+    def check_token(self):
+        if (self.identity.refresh and self.identity.expires_at <= time.time()):
```

Comment:
We have to load the identity to ensure all processes will have the latest state of it. This solution will be temporary until we fix the way we manage the identity and configuration.